packages/progress-bar/docs/ProgressBar.stories.tsx

```diff
@@ -0,0 +1,124 @@
+import React from "react"
+import { CATEGORIES } from "../../../storybook/constants"
+import { figmaEmbed } from "../../../storybook/helpers/figmaEmbed"
+import { ProgressBar } from "../index"
+
+export default {
+  title: `${CATEGORIES.components}/ProgressBar`,
+  component: ProgressBar,
+  parameters: {
+    docs: {
+      description: {
+        component: 'Import { ProgressBar } from "@kaizen/progress-bar"',
+      },
+    },
+  },
+}
+
+export const DefaultStory = _ => (
+  <ProgressBar
+    value={{ kind: "percentage", value: 25 }}
```

Comment:
This is what the "nested prop" with the "discriminated union type" mentioned earlier looks like in usage.

Reply:
Alternatively it could be:

for percentage: 

```
<ProgressBar
    variant="loading"
    value={100}
    mood="positive"
 />
 ```

 for a fraction:
 ```
<ProgressBar
    variant="loading"
    value={1}
    max={10}
    mood="positive"
 />
 ```
    
which looks a little neater, but my point stands about the type errors being much easier to read with the nesting, and also it's really not clear that the first one will render as a percentage (the other one is more self-documenting)